docs/content/wiring.md

```diff
@@ -259,19 +244,31 @@ The wordpress bundle declares a dependency on the `mysql` bundle, which we saw a
 install:
 - description: "Install Wordpress"
   helm:
-    name:
-      source: bundle.parameters.wordpress-name
+    name: {{ bundle.parameters.wordpress-name }} 
     chart: stable/wordpress
     replace: true
     set:
-      externalDatabase.database:
-        source: bundle.dependencies.mysql.parameters.database-name
-      externalDatabase.user:
-        source: bundle.dependencies.mysql.parameters.mysql-user
-      externalDatabase.password:
-        source: bundle.dependencies.mysql.outputs.mysql-password
+      externalDatabase.database: {{ bundle.dependencies.mysql.parameters.database-name }}
+      externalDatabase.user: {{ bundle.dependencies.mysql.parameters.mysql-user }}
+      externalDatabase.password: {{ bundle.dependencies.mysql.outputs.mysql-password }}
 ```
 
-When the install is executed for this bundle, the steps defined in the `mysql` bundle are completed first. Once those steps have run, any outputs defined are available. In this case, we want to use the `mysql-password` output from the `mysql` dependency. As the example YAML indicates, we do so with the declaration `source: bundle.dependencies.mysql.outputs.mysql-password`. The Porter runtime uses the `mysql` manifest to determine how to obtain the values from the dependency output and parameters.
+When the install is executed for this bundle, the steps defined in the `mysql` bundle are completed first. Once those steps have run, any outputs defined are available. In this case, we want to use the `mysql-password` output from the `mysql` dependency. As the example YAML indicates, we do so with the declaration `{{ bundle.dependencies.mysql.outputs.mysql-password }}`. The Porter runtime uses the `mysql` manifest to determine how to obtain the values from the dependency output and parameters.
 
-For more information on how dependencies are handled, refer to the [dependencies](/dependencies) documentation.
\ No newline at end of file
+For more information on how dependencies are handled, refer to the [dependencies](/dependencies) documentation.
+
+## Combining References
+
+It is possible to reference multiple parameters, credentials and/or outputs in a single place. You simply combine the expressions as follows:
```

Comment:
nit: Let's not use terms like "simply" in the docs. Bridget just [tweeted](https://twitter.com/bridgetkromhout/status/1116730265647370241) about why.

 As a replacement you can say "You can combine the expressions as follows"